exercises/concept/chaitanas-colossal-coaster/.docs/introduction.md

```diff
@@ -142,9 +151,9 @@ You can find the number of occurrences of an element in a list with the help of
 
 ## Finding the index of items
 
-`list.index()` will return the index number of the _first occurrence_ of an item passed in. If there are no occurrences, a `ValueError` is raised. If you don't need the exact position of an item and are only checking that it is present inside the list, the built-in `in` operator is more efficient.
+`list.index()` will return the index number of the _first occurrence_ of an item passed in. If there are no occurrences, a `ValueError` is raised. If the exact position of an item isn't needed, the built-in `in` operator is more efficient for checking simple membership.
```

Comment:
```suggestion
`list.index()` will return the index number of the _first occurrence_ of an item passed in. If there are no occurrences, a `ValueError` is raised. If the exact position of an item isn't needed, the built-in `in` operator is more efficient for checking if a list contains that value.
```

Reply:
Noticed a couple other edits needed, so did those in a self-edit that included these.  Going to "resolve" this.